hack/e2e/run.sh

```diff
@@ -104,18 +105,29 @@ if [[ $? -ne 0 ]]; then
 fi
 
 loudecho "Deploying driver"
+startSec=$(date +'%s')
 "${HELM_BIN}" upgrade --install "${DRIVER_NAME}" \
   --namespace kube-system \
   --set image.repository="${IMAGE_NAME}" \
   --set image.tag="${IMAGE_TAG}" \
   -f "${HELM_VALUES_FILE}" \
+  --wait \
   ./charts/"${DRIVER_NAME}"
 
 if [[ -r "${EBS_SNAPSHOT_CRD}" ]]; then
   loudecho "Deploying snapshot CRD"
   kubectl apply -f "$EBS_SNAPSHOT_CRD"
   # TODO deploy snapshot controller too instead of including in helm chart
 fi
+endSec=$(date +'%s')
+secondUsed=$(( (endSec-startSec)/1 ))
+# Set timeout threshold as 20 seconds for now, usually it takes less than 10s to startup
+if [ $secondUsed -gt $DRIVER_START_TIME_THRESHOLD ]; then
+  loudecho "Driver start timeout, test fail!"
```

Comment:
You should log here how long it took and what the threshold is, so we can see the gap immediately without reading the code.

Reply:
Updated